index.md

```diff
@@ -1,9 +1,34 @@
-## Infrastructure services
+Intended Audience: infrastructure team developers
 
-* [Yet Another SDR Handbook](yash)
+## infrastructure-integration-test
 
-* [The Stanford Digital Repository (SDR)](sdr/index)
+https://github.com/sul-dlss/infrastructure-integration-test)
+
+A collection of Capybara integration tests for the SDR.  Since it operates at the UI level and exercises a number of basic SDR functions that cross services, it can be a good walk through of some basic use cases in addition to providing regression tests.
+
+
+## Infrastructure service areas
+
+* [Stanford Digital Repository Software Services (SDR)](sdr/index)
+  * [SDR concepts and service interactions](sdr/concepts_and_interactions)
 * KSR
 * sul_pub
 * Sinopia
```

Comment:
i guess we're leaving this here for now.

Reply:
yeah, and i fleshed it out a bit in https://github.com/sul-dlss/infrastructure-team/pull/9 (can be broken out into separate pages if there's further elaboration)